src/unit/get.js

```diff
@@ -0,0 +1,76 @@
+define([
+	"./categories"
+], function( unitCategories ) {
+
+function stripPluralGarbage( data ) {
+	var aux, pluralCount;
+
+	if ( data ) {
+		aux = {};
+		for ( pluralCount in data ) {
+			aux[ pluralCount.replace( /unitPattern-count-/, "" ) ] = data[ pluralCount ];
+		}
+	}
+
+	return aux;
+}
+
+/**
+ * get( unit, form, cldr )
+ *
+ * @unit [String] The full type-unit name (eg. duration-second), or the short unit name
+ * (eg. second).
+ *
+ * FIXME
```

Comment:
Let's replace `FIXME` by adding a description for `@form` and `@cldr` attributes.